mesos/src/main/java/brooklyn/location/mesos/framework/marathon/MarathonLocation.java

```diff
@@ -161,6 +161,7 @@ private String getTaskName(Entity entity) {
         String planId = entity.config().get(BrooklynCampConstants.PLAN_ID);
```

Comment:
Here we don't have auto-generated value, so the logic is a bit different, Makes sense to reuse the logic, but at a higher level.